File: scala/src/main/scala/SlackAlphabet.scala
import java.util.Locale
import scala.annotation.tailrec

object SlackAlphabet extends App {
  println("Hello, world! Type in the text to translate.")

  @tailrec
  def run() {
    val input = scala.io.StdIn.readLine()
    input.toLowerCase(Locale.ENGLISH).toList.foreach { c =>
      if (c == ' ') print("  ")
      else if (c >= 'a' && c <= 'z') print(s":alphabet-white-$c: ")
    }
    println()
    run()
  }

  run()
}

File: python3/slack_alphabet.py
def _to_letter(c: str) -> str:
    return f":alphabet-white-{c}:"


def slackify(text: str) -> str:
    return " ".join(map(_to_letter, text))


def main():
    print(slackify(input("Type in the text to translate:")))


if __name__ == "__main__":
    main()
